src/screens/home/components/notificationCard/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { View, Image, TouchableHighlight, Text } from 'react-native'
+import type { FC } from 'react'
+import { type IComponentProps } from './notification-card.types'
+import { useNavigation } from '@react-navigation/native'
+
+const NotificationCard: FC<IComponentProps> = ({
+  photoURL,
+  title,
+  date,
+  redirect,
+  isRead = false
+}) => {
+  const navigation = useNavigation()
+
+  const redirectHandler = (): void => {
+    navigation.navigate(redirect)
+  }
+
+  return (
+    <TouchableHighlight onPress={redirectHandler}>
+      <View
+        className={`w-[375px] p-[16px] border-[0px] border-l-[5px] ${isRead ? 'border-[#FFF] bg-[#FFF]' : 'border-[#D83636] bg-[#EDEDED]'}`}
```

Comment:
Lets not use fixed values for for width since this wont allow this to work on all screen sizes. `w-full` should do the trick. Also, for padding 16 you can use `p-4`. For border-[0px] you can do `border-none`. The `border-l-[5px]` is fine since you don't have access to 5px. Regarding using hex values, please add the colors you need to the theme in `tailwind.config.js` so you could do something like: `bg-notification-bg` instead of #FFF.
```suggestion
        className={`w-full p-4 border-none border-l-[5px] ${isRead ? 'border-[#FFF] bg-notification-bg' : 'border-[#D83636] bg-[#EDEDED]'}`}
```

Reply:
In addition to the comment above, please also create the dark variant for the notification component. Here's the look and feel:

![image](https://github.com/victorstein/TGC/assets/11080740/39d49c76-e566-4f5c-a2ca-26c554f7828e)

Reply:
This can be done with the `dark:` class name. Please refer to other components